@donatello this breaks mint tests

```
(14/15) Running s3select tests ... FAILED in 0 seconds
{
  "name": "s3select:test_csv_input_quote_char",
  "function": "select_object_content(bucket_name, object_name, request)",
  "args": {
    "bucket_name": "s3select-test-bc6465bf-bca6-4948-8f35-e64b7d8108e1"
  },
  "duration": 228,
  "message": "Test test_5 unexpectedly failed with: S3 operation failed; code: MalformedXML, message: The XML provided was not well-formed or did not validate against our published schema. Check the service documentation and try again: unsupported QuoteCharacter '', resource: /s3select-test-bc6465bf-bca6-4948-8f35-e64b7d8108e1/20a2486c-df3f-4c36-a900-c3fcee91b26c, request_id: 17EC45F37E92AA5D, host_id: 1fc5fee7-5458-44f5-9068-a756dab35814, bucket_name: s3select-test-bc6465bf-bca6-4948-8f35-e64b7d8108e1, object_name: 20a2486c-df3f-4c36-a900-c3fcee91b26c",
  "error": "Traceback (most recent call last):\n  File \"/mint/run/core/s3select/csv.py\", line 39, in test_sql_api\n    data = client.select_object_content(bucket_name, object_name, sql_opts)\n  File \"/usr/local/lib/python3.10/dist-packages/minio/api.py\", line 601, in select_object_content\n    response = self._execute(\n  File \"/usr/local/lib/python3.10/dist-packages/minio/api.py\", line 440, in _execute\n    return self._url_open(\n  File \"/usr/local/lib/python3.10/dist-packages/minio/api.py\", line 423, in _url_open\n    raise response_error\nminio.error.S3Error: S3 operation failed; code: MalformedXML, message: The XML provided was not well-formed or did not validate against our published schema. Check the service documentation and try again: unsupported QuoteCharacter '', resource: /s3select-test-bc6465bf-bca6-4948-8f35-e64b7d8108e1/20a2486c-df3f-4c36-a900-c3fcee91b26c, request_id: 17EC45F37E92AA5D, host_id: 1fc5fee7-5458-44f5-9068-a756dab35814, bucket_name: s3select-test-bc6465bf-bca6-4948-8f35-e64b7d8108e1, object_name: 20a2486c-df3f-4c36-a900-c3fcee91b26c\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File \"/mint/run/core/s3select/./tests.py\", line 53, in main\n    test_csv_input_custom_quote_char(client, log_output)\n  File \"/mint/run/core/s3select/csv.py\", line 117, in test_csv_input_custom_quote_char\n    test_sql_api(f'test_{idx}', client, bucket_name,\n  File \"/mint/run/core/s3select/csv.py\", line 47, in test_sql_api\n    raise ValueError(\nValueError: Test test_5 unexpectedly failed with: S3 operation failed; code: MalformedXML, message: The XML provided was not well-formed or did not validate against our published schema. Check the service documentation and try again: unsupported QuoteCharacter '', resource: /s3select-test-bc6465bf-bca6-4948-8f35-e64b7d8108e1/20a2486c-df3f-4c36-a900-c3fcee91b26c, request_id: 17EC45F37E92AA5D, host_id: 1fc5fee7-5458-44f5-9068-a756dab35814, bucket_name: s3select-test-bc6465bf-bca6-4948-8f35-e64b7d8108e1, object_name: 20a2486c-df3f-4c36-a900-c3fcee91b26c\n",
  "status": "FAIL"
}```


It needs to be fixed, but it looks like this PR can break clients.

--- a/internal/s3select/csv/args.go
+++ b/internal/s3select/csv/args.go
@@ -27,8 +27,9 @@ import (
 )
 
 const (
-	none = "none"
-	use  = "use"
+	none   = "none"
+	use    = "use"
+	ignore = "ignore"
 
 	defaultRecordDelimiter      = "\n"
 	defaultFieldDelimiter       = ","
@@ -92,11 +93,22 @@ func (args *ReaderArgs) UnmarshalXML(d *xml.Decoder, start xml.StartElement) (er
 				}
 				switch tagName {
 				case "FileHeaderInfo":
-					args.FileHeaderInfo = strings.ToLower(s)
+					s = strings.ToLower(s)
+					if len(s) != 0 {
+						if s != none && s != use && s != ignore {
+							return errors.New("unsupported FileHeaderInfo")
+						}
+						args.FileHeaderInfo = s
+					}
+
 				case "RecordDelimiter":
-					args.RecordDelimiter = s
+					if len(s) != 0 {
+						args.RecordDelimiter = s
+					}
 				case "FieldDelimiter":
-					args.FieldDelimiter = s
+					if len(s) != 0 {
+						args.FieldDelimiter = s
+					}
 				case "QuoteCharacter":
 					if utf8.RuneCountInString(s) > 1 {
 						return fmt.Errorf("unsupported QuoteCharacter '%v'", s)
@@ -112,7 +124,9 @@ func (args *ReaderArgs) UnmarshalXML(d *xml.Decoder, start xml.StartElement) (er
 						return fmt.Errorf("unsupported QuoteEscapeCharacter '%v'", s)
 					}
 				case "Comments":
-					args.CommentCharacter = s
+					if len(s) != 0 {
+						args.CommentCharacter = s
+					}
 				default:
 					return errors.New("unrecognized option")
 				}

